packages/build/tests/constants/fixtures/account_id/plugin.js

```diff
@@ -0,0 +1,3 @@
+export const onPreBuild = function ({ constants: { ACCOUNT_ID } }) {
+  console.log(ACCOUNT_ID)
+}
```

Comment:
Is a test case missing in https://github.com/netlify/build/blob/main/packages/build/tests/constants/tests.js that would use that fixture?